nextflow.config

```diff
@@ -109,8 +109,28 @@ params {
     bamfiltering_savefilteredbams         = false // can include unmapped reads if --bamfiltering_retainunmappedgenomicbam specified
 
     // Metagenomic Screening
-    run_metagenomicscreening   = false
-    metagenomicscreening_input = 'unmapped' // mapped, all, unmapped -> mapped vs all specified in SAMTOOLS_FASTQ_MAPPED in modules.conf, unmapped hardcoded SAMTOOLS_FASTQ_UMAPPED
+    run_metagenomics_screening                                 = false
+    metagenomics_screening_input                               = 'unmapped' // mapped, all, unmapped -> mapped vs all specified in SAMTOOLS_FASTQ_MAPPED in modules.conf, unmapped hardcoded SAMTOOLS_FASTQ_UMAPPED
+    metagenomics_profiling_tool                                = null
+    metagenomics_profiling_database                            = null
+    metagenomics_profiling_krakenuniq_save_reads               = false
+    metagenomics_profiling_krakenuniq_save_read_classifications = false
+    metagenomics_profiling_krakenuniq_ram_chunk_size           = '16G'
+    metagenomics_profiling_kraken2_save_reads                  = false
+    metagenomics_profiling_kraken2_save_readclassification     = false
+    metagenomics_profiling_kraken2_save_minimizers             = false
+    metagenomics_profiling_malt_mode                           = 'BlastN'
+    metagenomics_profiling_malt_alignment_mode                 = 'SemiGlobal'
+    metagenomics_profiling_malt_min_percent_identity           = 85
+    metagenomics_profiling_malt_top_percent                    = 1
+    metagenomics_profiling_malt_max_queries                    = 100
+    metagenomics_profiling_malt_memory_mode                    = 'load'
+    metagenomics_profiling_malt_min_support_mode               = 'percent'
+    metagenomics_profiling_malt_min_support_percent            = 0.01
+    metagenomics_profiling_malt_min_support_reads              = 1
+    metagenomics_profiling_malt_sam_output                     = false
+    metagenomics_profiling_malt_save_reads                     = false
+    metagenomics_profiling_malt_group_size                     = 0
```

Comment:
Hmm, these are _very_ long parmaeter names...

I wonder if we can just call it `run_metagenomics` and `metagenomics_tool` `metagenomes_krakenuniq_save_reads` etc.